File: Project/Frontend/daariiou-photography-frontend/src/api/lib/models/picture.ts
/* tslint:disable */
/* eslint-disable */
import { Byte } from '@angular/compiler/src/util';
import { KindOfShooting } from './kind-of-shooting';
import { User } from './user';
export interface Picture {
  date?: string;
  isPublic?: null | string;
  koS?: null | KindOfShooting;
  koSid?: number;
  pid?: number;
  src?: null | Byte[];
  thumb?: null | string;
  uid?: null | number;
  uidNavigation?: null | User;
}

File: Project/Backend/daariiou_photography_backend/daariiou_photography_backend/Models/Picture.cs
using System;
using System.Collections.Generic;

#nullable disable

namespace daariiou_photography_backend.Models
{
    public partial class Picture
    {
        public int Pid { get; set; }
        public string Src { get; set; }
        public string IsPublic { get; set; }
        public DateTime Date { get; set; }
        public int? Uid { get; set; }
        public string Thumb { get; set; }
        public int KoSid { get; set; }

        public virtual KindOfShooting KoS { get; set; }
        public virtual User UidNavigation { get; set; }
    }
}
